### PR TITLE
Add an example of `neo4j-admin database upload` for Windows users

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

File: modules/ROOT/pages/tools/neo4j-admin/upload-to-aura.adoc [hunk 1]
```diff
@@ -176,3 +176,13 @@ Import progress
...
 Your data was successfully pushed to Aura and is now running.
 ----
+
+
+In Windows, do not forget to use double backslashes `//` in the path file.
+For example:
+
+[source, shell,role=nocopy]
+----
+bin/neo4j-admin database upload dbname --from-path=c://db-dump-file// --to-uri=<neo4j+s://your-aura-instance-id.databases.neo4j.io> --overwrite-destination=true
```
Comment (renetapopova) [inline modules/ROOT/pages/tools/neo4j-admin/upload-to-aura.adoc:186]:
```suggestion
bin\neo4j-admin database upload dbname --from-path=c:\\db-dump-file\\ --to-uri=<neo4j+s:\\your-aura-instance-id.databases.neo4j.io> --overwrite-destination=true
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1700.surge.sh

Updated pages:
- [Upload to Neo4j Aura](https://neo4j-docs-operations-1700.surge.sh/operations-manual/5/tools/neo4j-admin/upload-to-aura)

File: modules/ROOT/pages/tools/neo4j-admin/upload-to-aura.adoc [hunk 2]
```diff
@@ -176,3 +176,13 @@ Import progress
 .................... 100%
 Your data was successfully pushed to Aura and is now running.
 ----
+
+
+In Windows, do not forget to use double backslashes `//` in the path file.
```
Comment (renetapopova) [inline modules/ROOT/pages/tools/neo4j-admin/upload-to-aura.adoc:181]:
```suggestion
On Windows, the backslashes `\` in the file paths must be escaped with another backslash.
```

**Review by renetapopova — CHANGES_REQUESTED** [review]
On Windows they are backslashes.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1700.surge.sh

Updated pages:
- [Upload to Neo4j Aura](https://neo4j-docs-operations-1700.surge.sh/operations-manual/5/tools/neo4j-admin/upload-to-aura)

**Review by renetapopova — APPROVED** [review]
Looks good.

**Pull request merged**